cypress/integration/v2/blocks/team-member.spec.js

```diff
@@ -57,6 +61,241 @@ function switchDesign() {
 	] ) )
 }
 
+function styleTab( viewport, desktopOnly, registerBlockSnapshots ) {
+	cy.setupWP()
+	cy.newPage()
+	cy.addBlock( 'ugb/team-member' ).as( 'teamMemberBlock' )
+	const teamMemberBlock = registerBlockSnapshots( 'teamMemberBlock' )
+	cy.openInspector( 'ugb/team-member', 'Style' )
+
+	// General Tab
+	cy.collapse( 'General' )
+
+	desktopOnly( () => {
+		cy.adjust( 'Columns', 3 )
+		cy.get( '.ugb-team-member__item3' ).should( 'exist' )
+	} )
+	assertAligns( 'Align', '.ugb-inner-block', { viewport } )
+
+	// Container Tab
+	cy.collapse( 'Container' )
+
+	assertContainer( '.ugb-team-member__item', { viewport }, 'column%sBackgroundMediaUrl' )
+
+	range( 1, 4 ).forEach( idx => {
+		cy.setBlockAttribute( {
+			[ `image${ idx }Url` ]: Cypress.env( 'DUMMY_IMAGE_URL' ),
```

Comment:
image3Url is also set for tablet & mobile even if the columns set for tablet/mobile is only 2 (default). We can move the cy.adjust( 'Columns', 3 ) outside of desktopOnly.